Library/Formula/sqliteodbc.rb

```diff
@@ -0,0 +1,23 @@
+class Sqliteodbc < Formula
+  desc "SQLite ODBC driver"
+  homepage "http://www.ch-werner.de/sqliteodbc/"
+  url "http://www.ch-werner.de/sqliteodbc/sqliteodbc-0.9992.tar.gz"
+  sha256 "f5c6119cfb688fc3d2fbc500c6cfa796538b767c511be5ab90fb6805fcd289c3"
+
+  depends_on "sqlite"
+  depends_on "unixodbc"
+
+  def install
+    lib.mkdir
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+    File.symlink "#{lib}/libsqlite3odbc.dylib", "#{lib}/libsqlite3odbc.so"
```

Comment:
Can we use `lib.install_symlink "#{lib}/libsqlite3odbc.dylib" => "libsqlite3odbc.so"` rather than throwing out to `File` directly? Thanks.